### PR TITLE
Czech text correction

### PR DESCRIPTION
**Pull request by kolaron**
Pull request with few text corrections, which I spotted. There was few small changes, but it's my first pull request, so I hope it's not useless. 👍

### CODE REVIEW

File: cs/noswears/partials/intro.njk [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<p>Git je složitý: něco rozbít je jednoduché, ale přijít na to, jak z toho ven někdy téměř nemožné. Hledat něco v dokumentaci Gitu jde těžko <em>pokud tedy už nevíš název té věci, kterou hledáš</em>. </p>
+<p>Git je složitý: něco rozbít je jednoduché, ale přijít na to, jak z toho ven někdy téměř nemožné. Hledat něco v dokumentaci Gitu jde těžko, <em>pokud tedy už nevíš název té věci, kterou hledáš</em>. </p>
```
Comment (adaamz) [inline cs/noswears/partials/intro.njk:1]:
```suggestion
<p>Git je složitý: něco rozbít je jednoduché, ale přijít na to, jak z toho ven je někdy téměř nemožné. Hledat něco v dokumentaci Gitu jde těžko, <em>pokud tedy už nevíš název té věci, kterou hledáš</em>. </p>
```

File: cs/swears/partials/intro.njk [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<p>Git je složitý: něco posrat je jednoduché, ale přijít na to, jak z toho ven někdy téměř nemožné. Hledat něco v dokumentaci Gitu jde těžko <em>pokud tedy už nevíš název té věci, kterou hledáš</em>. </p>
+<p>Git je složitý: něco posrat je jednoduché, ale přijít na to, jak z toho ven někdy téměř nemožné. Hledat něco v dokumentaci Gitu jde těžko, <em>pokud tedy už nevíš název té věci, kterou hledáš</em>. </p>
```
Comment (adaamz) [inline cs/swears/partials/intro.njk:1]:
```suggestion
<p>Git je složitý: něco posrat je jednoduché, ale přijít na to, jak z toho ven je někdy téměř nemožné. Hledat něco v dokumentaci Gitu jde těžko, <em>pokud tedy už nevíš název té věci, kterou hledáš</em>. </p>
```

**Review by adaamz — CHANGES_REQUESTED** [review]

**Comment by ksylor** [comment]
Thanks so much for this @kolaron

I'm not sure if you think that the additional changes @adaamz suggested should be added to your PR? It would be great to figure that out before I merge this in. Let me know, thanks!

**Review by kolaron — COMMENTED** [review]
You can merge it, all additional changes by @adaamz are OK.

**Comment by ksylor** [comment]
Thanks so much @kolaron !!

**Pull request merged**